src/cmd/vm.rs

```diff
@@ -218,6 +207,7 @@ fn run_start(args: &ArgsStart) -> Result<()> {
     if let Some(extra_args) = &args.extra_args {
         options.append(&mut vec![extra_args]);
     }
+    options.append(&mut vec!["--display", "none"]);
```

Comment:
To unify the behavior with other DUTs, run `./betty.sh start` with the `--display none` option.